Source/WebCore/svg/SVGAngleValue.h

```diff
@@ -30,11 +30,14 @@ class SVGAngleValue {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     enum Type {
+        // FIXME: change the casing style of the enum type or naming of the Angle Value
```

Comment:
FIXMEs have a particular style, they need to be written as:

"FIXME: Sentence starts with uppercase and ends with a dot."

And then when there's a URL they're written as

"FIXME: We need to fix this (url)."

Reply:
fixed.